src/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$header-font-weight: 900;
+$header-font-size: 200%;
+$error-message-color: #ED4337;
+$header-left-color: #04529D;
+$header-right-color:  #FF2C06;
+$headerbar-header-left-color: #FFFFFF;
+$header-background: #466bc1;
+$button-hover-color: #3859a5;
+$button-padding: 12px;
+$side-bar-header-padding: 4px;
+$profile-pic-height: 185px;
+$profile-pic-width: 200px;
+$profile-pic-padding: 15px;
+$sidebar-background: #FFFFFF;
```

Comment:
In other pages where header is not used, the PC Prep is blue in color. But when we add header which is again in blue color, I made the heading there white.